### PR TITLE
feat: option to include only specific events

### PR DESCRIPTION
**Pull request by ConradKurth**
I would like to introduce this because when we send everything to customerio it create a ton of unique customers due to the unique volume of the amount of people coming to our site.

this will allow us to only allow specific events to go to customerio that we will include for email campaigns

### CODE REVIEW

**Comment by ConradKurth** [comment]
@Twixes Sorry, I have another one. Let me know what you think about this!

File: index.js [hunk 1]
```diff
@@ -25,13 +25,18 @@ export async function setupPlugin({ config, global }) {
         throw new RetryError('Service is down, retry later')
     }
 
+    const eventNames = (config.eventsToSend || '').split(',').filter(String)
```
Comment (Twixes) [inline index.js:28]: I don't think `.filter(String)` does anything? I think it should be `.filter(Boolean)` if you want to filter out non-empty elements.

**Review by Twixes — COMMENTED** [review]
Good idea, just got one Q regarding code :)

File: index.js (hunk 1 above)
Comment (ConradKurth) [reply index.js:28]: @Twixes So I will be the first to admit I am not the best javascript dev, but I did test this out and it does filter out the empty elements. I have tested it with the boolean way and that does work as well. Want me to do the boolean way??

Comment (Twixes) [reply index.js:28]: Oh, you're right, I forgot JS does a conversion to boolean here anyway. No need to change then.

**Review by Twixes — APPROVED** [review]

**Pull request merged**